### PR TITLE
fix: update `zdharma` to `zdharma-continuum`

### PR DESCRIPTION
**Pull request by findNextStep**
make readme.md more useful

see https://github.com/zdharma-continuum/zinit/issues/2

### CODE REVIEW

**Review by alichtman — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1583,9 +1583,12 @@ If you load completions using `wait''` Turbo mode then you can add
 `atinit'zicompinit'` to syntax-highlighting plugin (which should be the last
 one loaded, as their (2 projects, [z-sy-h](https://github.com/zsh-users/zsh-syntax-highlighting) &
 [f-sy-h](https://github.com/zdharma-continuum/fast-syntax-highlighting))
-documentation state), or `atload'zicompinit'` to last
-completion-related plugin. `zicompinit` is a function that just runs `autoload compinit; compinit`, created for convenience. There's also `zicdreplay` which
-will replay any caught compdefs so you can also do: `atinit'zicompinit; zicdreplay'`, etc. Basically, the whole topic is the same as normal `compinit` call,
+
+ documentation state), or `atload'zicompinit'` to last
```
Comment (alichtman) [inline README.md:1587]: Remove this extra line

File: README.md [hunk 2]
```diff
@@ -1652,7 +1655,8 @@ source smaller helper scripts or dependency libraries (for example, the prompt `
 and there are very few solutions to that, which are demanding (e.g. specifying all helper files in plugin load
 command and investigating updates to the plugin – in Zinit case: by using `compile` ice-mod).
 
-![image](https://raw.githubusercontent.com/zdharma-continuum/zinit/images/mod-auto-compile.png)
+  ![image](https://raw.githubusercontent.com/zdharma-continuum/zinit/images/mod-auto-compile.png)
```
Comment (alichtman) [inline README.md:1658]:
```suggestion
![image](https://raw.githubusercontent.com/zdharma-continuum/zinit/images/mod-auto-compile.png)
```

File: README.md [hunk 3]
```diff
@@ -1760,6 +1764,8 @@ zinit as"null" wait"1" lucid for \
             zdharma-continuum/zsh-diff-so-fancy \
     sbin"git-url;git-guclone" make"GITURL_NO_CGITURL=1" \
             zdharma-continuum/git-url
+# NOTICE: zdharma/zsh-diff-so-fancy is still not available
```
Comment (alichtman) [inline README.md:1767]:
```suggestion
```

Comment (alichtman) [reply README.md:1767]: It won't be available in the future either. I don't want to distribute it -- it doesn't make sense to me

**Review by alichtman — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (findNextStep) [reply README.md:1767]: maybe we need remove this plugin in code in same time

Comment (alichtman) [reply README.md:1767]: I didn't port it over. That is "removing it."

**Review by alichtman — APPROVED** [review]

File: README.md (hunk 3 above)
Comment (alichtman) [reply README.md]: oh, I see what you meant

**Pull request merged**